src/database/migrations/z1631541043-update-inflow.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    if (process.env.NODE_ENV === 'test') {
+      return;
+    }
+
+    //add column
+    await queryInterface.addColumn('inflow', 'contractAddress', {
+        type: Sequelize.STRING(44),
+        allowNull: false,
+        defaultValue: '0',
+    });
+
+    await queryInterface.addColumn('inflow', 'asset', {
+      type: Sequelize.STRING(4),
+      defaultValue: 'cUSD',
+      allowNull: false,
+    });
+
+    await queryInterface.addColumn('inflow', 'value', {
+      type: Sequelize.DECIMAL(29), // max 9,999,999,999 - plus 18 decimals
+      allowNull: false,
+      defaultValue: 0,
+    });
+
+    // populate new column
+    const query = `
+      UPDATE inflow
+      SET "contractAddress" = community."contractAddress", "value" = "amount"
+      FROM (SELECT "contractAddress", "publicId" FROM community) community
+      WHERE community."publicId" = "communityId"`;
+
+    await queryInterface.sequelize.query(query, {
+      raw: true,
+      type: Sequelize.QueryTypes.UPDATE,
+    });
+
+    // remove old column
+    await queryInterface.removeColumn(
+      'inflow',
+      'communityId'
+    );
```

Comment:
I'm not sure if you need to remove a constraint first. Just writing here, so I don't forget.

Reply:
Also, quick thought (also to not forget). I see you didn't change yet. This change requires updating the triggers.

Reply:
I didn't have this problem running locally, but, just to make sure, I added the "removeConstraint" before removing the column.
And now, I updated the triggers